apps/governance/src/routes/proposals/components/proposal/proposal-change-details.tsx

```diff
@@ -61,15 +64,26 @@ export const ProposalChangeDetails = ({
     }
     case 'NewMarket': {
       if (proposal.id) {
-        details = <ProposalMarketData proposalId={proposal.id} />;
+        let marketId = proposal.id;
+
+        // TODO: when https://github.com/vegaprotocol/vega/issues/11005 gets merged
```

Comment:
Do we have a ticket for this? It'd be good to have it nailed to the board so we won't forget.

Reply:
Done